ddp-workspace/projects/ddp-pancan/src/app/components/activity/activity.component.ts

```diff
@@ -29,9 +66,13 @@ export class ActivityComponent extends ActivityRedesignedComponent {
     }
 
     get hasNoEmptyNestedActivity(): boolean {
```

Comment:
Honestly, I am a bit confused with the method name.

Does it mean 'There are some not empty activities' or most likely 'There are not any empty activity' ? 

May be rename it something like 'allNestedActivitiesAreEmpty' / 'allNestedActivitiesAreNotEmpty' / 'allNestedActivitiesHaveAnswers' - or kind of ?

Reply:
It means "there are not any empty nested activities". I think `allNestedActivitiesHaveAnswers` sounds better